main.nf

```diff
@@ -2605,6 +2606,64 @@ process StrelkaBP {
 
 vcfStrelkaBP = vcfStrelkaBP.dump(tag:'Strelka BP')
 
+// STEP MSISENSOR.1 - SCAN
+
+// Scan reference genome for microsattelites
+process msisensorScan {
+    label 'cpus_1'
+    label 'memory_max'
+    // memory '20 GB'
+
+    tag {fasta}
+
+    input:
+    file(fasta) from ch_fasta
+    file(fastaFai) from ch_fai
+
+    output:
+    file "microsatellites.list" into msi_scan_ch
+
+    when: 'msisensor' in tools
+
+    script:
+    """
+    msisensor scan -d ${fasta} -o microsatellites.list
+    """
+}
+
+// STEP MSISENSOR.2 - SCORE
+
+// Score the normal vs somatic pair of bams
+
+process msisensor {
+    label 'cpus_4'
+    label 'memory_max'
+    // memory '10 GB'
+
+    tag {idSampleTumor + "_vs_" + idSampleNormal}
+
+    publishDir "${params.outdir}/MSI/${idSampleTumor}_vs_${idSampleNormal}/msisensor", mode: params.publishDirMode
```

Comment:
```suggestion
    publishDir "${params.outdir}/MSIsensor/${idSampleTumor}_vs_${idSampleNormal}/msisensor", mode: params.publishDirMode
```

Reply:
Mmmh but there’s 2 dirs caled msisensor. The first lne was equivalent to variant calling. We can put it there aswell I suppose.

Reply:
Oooh, I see, sorry about that, let me correct my suggestion then

Reply:
what I meant below is that all the other tools have their own nesting dir like `publishDir "${params.outdir}/VariantCalling` etc. That's why I originally wrote only MSI. Shouldn't we put msisensor in one of those?

Reply:
okok solved